pulsar-zookeeper-utils/src/main/java/org/apache/pulsar/zookeeper/LocalBookkeeperEnsemble.java

```diff
@@ -174,7 +175,22 @@ private void runBookies(ServerConfiguration baseConf) throws Exception {
             bsConfs[i].setAllowLoopback(true);
             bsConfs[i].setGcWaitTime(60000);
 
-            bs[i] = new BookieServer(bsConfs[i], NullStatsLogger.INSTANCE);
+            try {
+                bs[i] = new BookieServer(bsConfs[i], NullStatsLogger.INSTANCE);
+            } catch (InvalidCookieException e) {
+                // InvalidCookieException can happen if the machine IP has changed
```

Comment:
Is it limited to execution only for standalone start? I see the start() also calls this method.

Reply:
The `start()` is just used within unit tests, so it's safe to clear it as well. It's not used when launching the bookie with `bin/pulsar bookie`